cmd/fluxd/main.go

```diff
@@ -97,7 +99,15 @@ func (set stringset) has(possible string) bool {
 }
 
 func main() {
-	// Flag domain.
+	// --- Flags ---
+
+	// Generally we want to allow configuration to be supplied in a
+	// config file OR in the environment OR via command-line
```

Comment:
I thought we agreed on not accepting env variables. Is this simply not up-to-date?

Reply:
Correct: not up to date.